### DISCUSSION TITLE
Can anyone confirm this Axios memory leak.

### DISCUSSION THREAD

**Discussion by ekerner-com** [axios/axios#5640] [General]
I observed this in Chrome on Mac (so Safari essentially).
I haven't tested in any other environment.

**Every call with Axois adds 4 event listeners and they never get cleaned up**.
This inflates memory consumption over time and eventually causes a mem crash.

**To reproduce**
Leave a page open with an interval running an Axios call every n seconds.

**To observe**
Open the performance monitor and make sure that the JS Heap and Event Listeners metrics are selected.

![image](https://user-images.githubusercontent.com/5393913/230735923-88c82d5d-dc69-4007-8c31-c2e36794f590.png)

I tried the latest and older versions with the same result.

**Comment by jasonsaayman** [comment]
Closing as we need a propper re-creation with a repo etc. If you can give me this I would be happy to take a look
